react/features/overlay/components/OverlayContainer.js

```diff
@@ -1,12 +1,38 @@
-/* global APP */
-
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
+import PageReloadFilmStripOnlyOverlay from './PageReloadFilmStripOnlyOverlay';
 import PageReloadOverlay from './PageReloadOverlay';
+import SuspendedFilmStripOnlyOverlay from './SuspendedFilmStripOnlyOverlay';
 import SuspendedOverlay from './SuspendedOverlay';
+import UserMediaPermissionsFilmStripOnlyOverlay
+    from './UserMediaPermissionsFilmStripOnlyOverlay';
 import UserMediaPermissionsOverlay from './UserMediaPermissionsOverlay';
 
+declare var APP: Object;
+declare var interfaceConfig: Object;
+
+/**
+ * Maps the overlay component that have to be displayed depending on the
+ * filmstrip only mode.
+ *
+ * @type {object}
+ */
+const filmStripOnlyModeToComponent = {
+    pageReload: {
+        filmStripOnly: PageReloadFilmStripOnlyOverlay,
```

Comment:
OK. I did it and it doesn't look good to me. I like the current version better. 

Since you said you don't have strong argument I prefer to leave it as it is now.